Jupyterlab nvdashboard v0 3 0

owenlamont opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Making a 0.3.0 release - explicitly made this version depend on jupyter lab 1 and set the maximum bokeh version to less than 2.

I have a follow up [PR](https://github.com/conda-forge/jupyterlab-nvdashboard-feedstock/pull/6) to release version 0.3.1 and make jupyter lab 2 a requirement.

owenlamont committed:
Updated meta.yaml to use 0.3 and tweaked the
requirements

owenlamont committed:
MNT: Re-rendered with conda-build 3.18.11, conda-smithy 3.7.3, and conda-forge-pinning 2020.06.06.02.30.37

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

owenlamont commented:
I built and installed this recipe locally on my laptop and added it to a new conda environment. It correctly installed jupyter lab 1 and appeared to be running correctly. I don't know whether now I've added jupyter lab itself as a requirement whether that makes some of the other requirements redundant.

jakirkham reviewed (approved):
LGTM. @jacobtomlinson @rjzamora, would either of you be able to take a look? 🙂

jakirkham commented:
If there are issues Rick and Jacob, please let us know.
